dist/common/scripts/scylla_util.py

```diff
@@ -1008,11 +1008,14 @@ def __load(self):
         self._cfg.optionxform = str
         self._cfg.readfp(f)
 
+    def __fmt(self, val):
+        return '{}="{}"' if ' ' in val else '{}={}'
```

Comment:
Let's check for other whitespace characters too like tabs for extra safety?

```python
need_quotes = any([ch.isspace() for ch in val])
return '{}="{}"' if need_quotes else '{}={}'
```

Reply:
@penberg Applied `need_quotes` to new version.